news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,183 @@
+
+As every month we keep you up to date with the changes to the Ranking Criteria. Additionally we list open proposals so you can give your feedback to them! Below a list of the new proposals since the last issue of the Mappers' Report.
+
+### All Modes
+
+- **[Treat official romanisation above RC standardisation](https://osu.ppy.sh/community/forums/topics/1095833)**
+- **[Avoid spoilers in maps](https://osu.ppy.sh/community/forums/topics/1083667)**
+- **[Clear metadata rules and add guidelines regarding mashups](https://osu.ppy.sh/community/forums/topics/1108054)**
+
+### osu!catch
+
+- **[Remove "clear dashes" guideline on Platter difficulties](https://osu.ppy.sh/community/forums/topics/1118458)**
+
+### osu!mania
+
+- **[Change some duplicated phrases in 7K hard guidelines](https://osu.ppy.sh/community/forums/topics/1112674)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+New Beatmap Nominators have arrived! As with every month, we would like to show you which people you can request your pending maps to, just make sure to request them to the Beatmap Nominators of the right game mode. Shown below are the lists of all additions and departures of this month!
+
+Before we get to them however, we have some important announcements to make regarding the so-called "care package" which has had its first test run some months ago with osu!taiko, along with a change regarding Beatmap Nominator applications and evaluations.
+
+### Activity lenience for returning Beatmap Nominators
+
+Have you ever been a Beatmap Nominator before and would like to make your return? Then we have great news! In the [February issue of the Mappers' Report](https://osu.ppy.sh/home/news/2020-02-28-mappers-report-february) we mentioned that we applied a "care package" for osu!taiko that gave past Beatmap Nominators an easier way to make their return. Now the time has come where we will apply it to all game modes, starting with this announcement!
+The following requirements apply for specific activity leniency:
+
+- **You must have been a member of the Beatmap Nominators in the past**. This applies back to the date when the Beatmap Nominators were first created. (Past BAT times). This means that you will only need **two months** of modding activity in order to apply!
+- **You must have been a member of the Beatmap Nominators in the past year**. Additionally you must have left "on good terms". "On good terms" means that you have passed your last evaluation. If both apply to you, you will only need **one month** of modding activity in order to apply!
+
+This has been tested on leaving Beatmap Nominators for a few months already, so you may already have this leniency if you've left recently.
+We've got your interest? Great! When you are sure that the activity lenience can be applied to you, contact any member of the Nomination Assessment Team! (Preferably of your game mode, because they will most likely know if you were a Beatmap Nominator in the past.)
```

Comment:
```suggestion
We've got your interest? Great! When you are sure that the activity lenience can be applied to you, contact any of the Nomination Assessment Team members! (Preferably of your game mode, because they will most likely know if you were a Beatmap Nominator in the past.)
```